doc/source/io.rst

```diff
@@ -4329,10 +4329,10 @@ a ``TableCreationError`` if the destination table already exists.
 
 .. note::
 
-   If the ``if_exists`` argument is set to ``'append'``, the destination dataframe will
-   be written to the table using the defined table schema and column types. The
-   dataframe must match the destination table in column order, structure, and
-   data types.
+   If the ``if_exists`` argument is set to ``'append'``, the dataframe will be written
+   to the destination table using the defined table schema and column types. The
+   dataframe must match the destination table in column order, structure, data types,
```

Comment:
put column order, structure, data types, mode in double back-ticks. (NULLABLE can be italiized)